spec/features/reviews/user_can_create_review_spec.rb

```diff
@@ -13,7 +13,6 @@
       visit "/shelters/#{shelter_1.id}"
 
       expect(page).to_not have_content(title)
-      expect(page).to_not have_content(rating)
```

Comment:
Had to remove this requirement because the total rating now shows on the show page in another section. Possibility to refactor this test into a within if we want.